app/views/groups/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="bg-primary p-2">
```

Comment:
* Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements. As your color here is different